jobs/pacman-tenable-enricher/src/main/java/com/tmobile/cso/pacman/tenable/Main.java

```diff
@@ -66,13 +66,13 @@ public static void execute(Map<String, String> params) {
         int days = Integer.parseInt(params.get("days"));
         if (jobHint.equals("tenable_vm_vulnerability")) {
             errorInfo = TenableVMVulnerabilityDataImporter.getInstance().execute(days);
-            if (!errorInfo.isEmpty() && errorInfo.get("error") != null) {
-                log.warn("Job executed with some errors -> {}", errorInfo);
+            if (!errorInfo.isEmpty() && errorInfo.get("errors") != null && ((List<Map<String, Object>>) errorInfo.get("errors")).size() > 0) {
+                log.error("Job executed with some errors -> {}", errorInfo);
             } else {
                 log.info("Job executed successfully");
             }
         } else {
-            log.warn("Job hint is not supplied");
+            log.error("Job hint is not supplied");
```

Comment:
Why is an error? It means that there is not data returned but it's a valid response from open search?

Reply:
This goes not from OS, it hoes from job definition, as a param. So if it's not provided then job was not configured properly during installation (see screenshot)
![image](https://github.com/PaladinCloud/CE/assets/133698330/c8acc1e5-e456-485a-ae1d-1ebbc48da9d2)